hier_config/workflow.py

```diff
@@ -0,0 +1,98 @@
+from collections.abc import Iterable
+from logging import getLogger
+
+from .model import TagRule
+from .root import HConfig
+
+logger = getLogger(__name__)
+
+
+class WorkflowRemediation:
+    """A host object is a convenient way to loading host inventory
+    items into a single object.
+
+    The default is to load "hostname", "os", and "options" to the host object,
+    however, it can easily be extended for developer needs.
+
+    .. code:: python
+
+        import yaml
+        from hier_config.host import Host
+
+        options = yaml.load(open("./tests/fixtures/options_ios.yml"), loader=yaml.SafeLoader())
+        host = Host("example.rtr", "ios", options)
+
+        # Example of loading running config and generated configs into a host object
+        host.load_running_config_from_file("./tests/files/running_config.conf)
+        host.load_generated_config_from_file("./tests/files/generated_config.conf)
+
+        # Example of creating a remediation config without a tag targeting specific config
+        host.remediation_config()
+
+        # Example of creating a remediation config with a tag ("safe") targeting a specific config.
+        host.remediation_config_filtered_text({"safe"}, set()})
+    """
```

Comment:
Is this docstring still correct? I noticed that the host.py is being removed as part of this PR.